prototype/link/scripts/preprocessing_exploration/README.md

```diff
@@ -12,15 +12,15 @@ The main scripts that should be run in a sequence (can be run for multiple runs
 
 *Input/s*: functional run. e.g. `sbatch ./scripts/preprocessing_exploration/prep_raw_data_explore.sh 01`
 *Output/s*: Confound files in $SUBJ_DIR/analysis/firstlevel/Confounds/ for different preprocessing parameters of a run
 
 `preprocessing_exploration.sh`: Performs a FEAT analysis for all of the parameters specified.
-For each different type of preprocessing setting (listed in the variable `analysis_types`) this performs a FEAT analysis. A set of default parameters are defined (starting in the section `Set the default values`) and each analysis changes parameters relative to those parameters. The `default` analyses are those with no parameters changed are reported in text as the chosen parameters. These parameters are then used to create fsf files with the appropriate parameters and then the FEAT analysis is run.  
+For each different type of preprocessing setting (listed in the variable `analysis_types`) this performs a FEAT analysis. A set of default parameters are defined (starting in the section `Set the default values`) and each analysis changes parameters relative to those defaults. The `default` analyses, which with no parameters changed, are reported in text as the chosen parameters. These parameters are then used to create fsf files with the appropriate parameters and then the FEAT analysis is run.  
 For some parameters you can change the value with the provided functionality (e.g. smoothing could be changed to any value fsl recognizes) but for adding new types of parameter searches you might need to create a new if statement in the section starting `# Change the parameters`.
-This performs the analysis two ways, it first runs the analysis with the data for a run or pseudorun as is and then second reruns it with a sliced version. The sliced data is created by `slice_data.m` which takes the timing file and creates new functional data, confound files and timing data for only the included blocks plus rest. This type of analysis is more equivalent to the way that analyses are ultimately run in the infant_neuropipe. Hence this is the analysis used as the default and is what is reported in the text.
+This performs the analysis two ways, it first runs the analysis with the data for a run or pseudorun as is and then second reruns it with a sliced version. The sliced data is created by `slice_data.m` which takes the timing file and creates new functional data, confound files and timing data for only the included blocks plus rest. This type of analysis is more equivalent to the way that analyses are ultimately run in the infant_neuropipe. Hence this is the analysis used as for subsequent consideration and is what is reported in the text.
```

Comment:
My rewrite: "This performs the analysis in two ways. In the first, it runs the analysis with the data for a run or pseudorun as is (using all of the collected TRs). In the second, the analysis is rerun with a 'sliced' version. The sliced data is created by `slice_data.m` which takes the timing file and creates new functional data, confound files and timing data for only the included blocks plus rest. This second type of analysis is more equivalent to the way that analyses are ultimately run in the infant_neuropipe. Hence, this is the analysis used for subsequent consideration and is what is reported in the text."

Reply:
Yeah I rewrote that section too thanks